.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,124 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: >-
+          EE name if it is community-ee-base or community-ee-minimal. Example: community-ee-base
+        required: true
+      ansible-core-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+      ee-version:
+        description: >-
+          EE Release Version. Example: 1
+        required: true
+      tag:
+        description: >-
+          Whether to tag the image as latest.
+        type: boolean
+        required: false
+        default: false
+
+env:
+  EE_VERSION: ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+
+jobs:
+  build:
+    name: Build Execution Environment ${{ inputs.ansible-core-version }}-${{ inputs.ee-version }}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+      packages: write
+
+    steps:
+      - name: Check out images
+        uses: actions/checkout@v4
+        with:
+          repository: ansible-community/images
+          ref: main
+          path: image
+
+      - name: Check out eercheck repo for testing
+        uses: actions/checkout@v4
+        with:
+          repository: anweshadas/eercheck
+          ref: main
+          path: eercheck
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install ansible-builder
+          python3 -m pip install setuptools
+          sudo apt install podman --yes
+          pwd
+          ls -l
+
+      - name: Pull the latest fedora image
+        run: |
+          podman pull fedora:latest
```

Comment:
If the goal of this step is to pull the image used in later by ansible-builder, you should probably specify `quay.io/fedora/fedora:latest`.
Current `fedora:latest` is forwarded to [`registry.fedoraproject.org/fedora:latest`](https://registry.fedoraproject.org/repo/fedora/tags/) by [shortnames](https://github.com/anweshadas/images/actions/runs/9039557300/job/24842585526#step:6:13), but it is probably out of date and different from the image specified in actual `execution-envioromnent.yml`, so the image pulled here will not be used.

Reply:
Thank you, updated.